macos/FluentUI/Button/Button.swift

```diff
@@ -374,7 +410,13 @@ open class Button: NSButton {
 
 	open override var intrinsicContentSize: CGSize {
 		let superSize = super.intrinsicContentSize
-		return CGSize(width: superSize.width,
+		var trailingImageAdjustment: CGFloat = 0
```

Comment:
can make this a let (prefer immutability) and avoid assigning it an initial value by assigning the value to 0 in an else block